test/start-stop-olm.sh

```diff
@@ -63,15 +63,33 @@ if [ $cmd == install ]; then
     # was successfull
     set +e
     echo "Installing OLM..."
-    if ! ${BIN_DIR}/operator-sdk olm install --version=${OLM_VERSION} --verbose --timeout=5m 2>&1 ; then
-        if ! ${BIN_DIR}/operator-sdk olm status 2>&1 ; then
-            echo "OLM installation failed!!!"
-            ${KUBECTL} get all -n olm
-            exit 1
+    attempt=0
+    while true; do
+        if ${BIN_DIR}/operator-sdk olm install --version=${OLM_VERSION} --verbose --timeout=5m 2>&1 &&
+               ${BIN_DIR}/operator-sdk olm status 2>&1 ; then
+            break
+        else
+            echo "OLM installation failed, attempt #$attempt!!!"
```

Comment:
This worked:

https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/job/pmem-csi/job/PR-1054/7/artifact/joblog-jenkins-pmem-csi-PR-1054-7-test-1.19.log

```
I1207 15:02:27.473497    1028 exec.go:72] "EnsureDeployment/stderr: + true" command="test/start-stop-olm.sh"
I1207 15:02:27.473543    1028 exec.go:72] "EnsureDeployment/stderr: + /mnt/workspace/pmem-csi_PR-1054/_work/bin/operator-sdk olm install --version=v0.18.3 --verbose --timeout=5m" command="test/start-stop-olm.sh"
I1207 15:02:27.706227    1028 exec.go:72] "EnsureDeployment/stdout: time=\"2021-12-07T15:02:27Z\" level=debug msg=\"Debug logging is set\"" command="test/start-stop-olm.sh"
I1207 15:02:28.113875    1028 exec.go:72] "EnsureDeployment/stdout: time=\"2021-12-07T15:02:28Z\" level=info msg=\"Fetching CRDs for version \\\"v0.18.3\\\"\"" command="test/start-stop-olm.sh"
I1207 15:02:28.113923    1028 exec.go:72] "EnsureDeployment/stdout: time=\"2021-12-07T15:02:28Z\" level=info msg=\"Fetching resources for resolved version \\\"v0.18.3\\\"\"" command="test/start-stop-olm.sh"
I1207 15:02:28.979503    1028 exec.go:72] "EnsureDeployment/stdout: time=\"2021-12-07T15:02:28Z\" level=fatal msg=\"Failed to install OLM version \\\"v0.18.3\\\": failed to get resources: failed to fetch resources: request failed: failed GET 'https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.3/olm.yaml': unexpected status code 503, expected 200: \\ufeff<?xml version=\\\"1.0\\\" encoding=\\\"utf-8\\\"?><Error><Code>ServerBusy</Code><Message>Egress is over the account limit.\\nRequestId:82405ad7-a01e-0070-7f7b-eb9b92000000\\nTime:2021-12-07T15:02:28.9529254Z</Message></Error>\"" command="test/start-stop-olm.sh"
I1207 15:02:28.982278    1028 exec.go:72] "EnsureDeployment/stderr: + echo 'OLM installation failed, attempt #0!!!'" command="test/start-stop-olm.sh"
I1207 15:02:28.982302    1028 exec.go:72] "EnsureDeployment/stdout: OLM installation failed, attempt #0!!!" command="test/start-stop-olm.sh"
...
I1207 15:03:32.060100    1028 exec.go:72] "EnsureDeployment/stdout: OLM installation sucessful after 1 retries!!!" command="test/start-stop-olm.sh"
````